src/configuration/config.js

```diff
@@ -10,6 +10,15 @@ import { alaskaConfig } from './regions/alaska';
 export const betaIdentifyEndpoint = 'https://api.resilientcoasts.org/beta/identify/';
 export const prodIdentifyEndpoint = 'https://api.resilientcoasts.org/prod/identify/';
 export const mapConfig = {
+  basemaps: {
+    'Dark Gray': 'ArcGIS:DarkGray',
+    // eslint-disable-next-line quote-props
+    'Imagery': 'ArcGIS:Imagery',
+    // eslint-disable-next-line quote-props
+    'Streets': 'ArcGIS:Streets',
+    // eslint-disable-next-line quote-props
+    'Topographic': 'ArcGIS:Topographic'
```

Comment:
You could just remove the quotes from these three and get rid of the linter comments. I'm fine either way though.

Reply:
yes please don't disable lint to get it too pass...